mozjs/build.rs

```diff
@@ -172,7 +172,11 @@ fn build_jsapi(build_dir: &Path) {
             // For now we also support mozilla build
             PathBuf::from(moz_build)
         } else {
-            panic!("MozTools not found!");
+            panic!(
+                "MozTools or MozillaBuild not found!\n
+Follow instructions on: https://github.com/servo/mozjs?tab=readme-ov-file#windows
+            "
+            );
```

Comment:
```suggestion
            panic!(
                "MozTools or MozillaBuild not found!\n \
                 Follow instructions on: https://github.com/servo/mozjs#windows "
            );
```

I think that the slash escapes leading whitespace.